crates/sui-types/src/messages.rs

```diff
@@ -1203,6 +1204,89 @@ impl InputObjectKind {
         }
     }
 }
+
```

Comment:
Why the move?

Reply:
it was needed to be able to do genesis from the sui-config crate